spec/models/ticket_spec.rb

```diff
@@ -43,11 +59,29 @@
       expect(ticket.bought?(user)).to eq(true)
     end
 
-    it 'returns true if the user has bought this ticket' do
+    it 'returns false if the user has not bought this ticket' do
       expect(ticket.bought?(user)).to eq(false)
     end
   end
 
+  describe '#paid?' do
+    let(:ticket_purchase) { create(:ticket_purchase, user: user, ticket: ticket) }
+
+    context 'user has paid' do
+      before { ticket_purchase.update_attributes(paid: true) }
+
+      it 'returns true' do
+        expect(ticket.paid?(user)).to eq(true)
+      end
+    end
+
+    context 'user has not paid' do
+      it 'returns false' do
+        expect(ticket.paid?(user)).to eq(true)
```

Comment:
Sorry, went a little overboard with copy and paste :stuck_out_tongue_closed_eyes: The method in question is:
```Ruby
def paid?(user)
    ticket_purchases.where(user_id: user.id, paid: false).count == 0
end
```

So, if `ticket_purchases` is empty, its count will be 0 and the test passes. I have changed the function on the assumption that between a given `ticket` and a `user` there can be only one `ticket_purchase`. I hope it makes sense.